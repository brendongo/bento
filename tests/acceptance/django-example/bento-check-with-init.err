Running Bento checks...


⚠ 1 finding(s) in 9.74 s
◦ To suppress all findings, run $ bento archive.
