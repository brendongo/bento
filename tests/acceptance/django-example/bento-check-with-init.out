r2c.jinja jinjalint-missing-meta-content-type https://bento.dev/checks/jinja/missing-meta-content-type/
     > app/templates/app/layout.html:1
     = HTML missing a meta content-type declaration may result in content
       misinterpretation in certain browsers, especially when using the
       'file://' protocol, and thus XSS. Include a meta content-type like '<meta
       http-equiv="Content-Type" content="text/html; charset=UTF-8">' to avoid
       misinterpretation.

r2c.jinja jinjalint-recursion-error
     > readme.html:0
     = Parse error: maximum recursion depth exceeded

